eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -343,6 +345,9 @@ object NodeParams extends Logging {
     require(Features.knownFeatures.map(_.mandatory).intersect(pluginFeatureSet).isEmpty, "Plugin feature bit overlaps with known feature bit")
     require(pluginFeatureSet.size == pluginMessageParams.size, "Duplicate plugin feature bits found")
 
+    val interceptOpenChannelPlugins = pluginParams.collect { case p: InterceptOpenChannelPlugin => p }
+    require(interceptOpenChannelPlugins.size <= 1, s"More than one plugin found that intercepts open channel messages: ${interceptOpenChannelPlugins.map(_.getClass.getSimpleName).mkString(", ")}")
```

Comment:
nit: I'd give slightly more concrete steps for the node operator to fix the issue:
```suggestion
    require(interceptOpenChannelPlugins.size <= 1, s"At most one plugin is allowed to intercept channel open messages, but multiple such plugins were registered: ${interceptOpenChannelPlugins.map(_.getClass.getSimpleName).mkString(", ")}. Disable conflicting plugins and restart eclair.")
```

Reply:
fixed in [916da34](https://github.com/ACINQ/eclair/pull/2552/commits/916da349f31f465d4bb9b9dafec868da6cf8a34d)